docs/source/production.rst

```diff
@@ -130,6 +130,11 @@ When trying to use the `dbt templater`_, uncomment the
 ``additional_dependencies`` to install the extras.
 This is equivalent to running ``pip install <dbt-adapter> sqlfluff-templater-dbt``.
 
+You can specify the verion of ``dbt-adapter`` used in `pre-commit`_
+E.g. ``additional_dependencies : ['dbt-bigquery==1.0.0', 'sqlfluff-templater-dbt']``
```

Comment:
Looks a bit messy when I run the site:

![image](https://user-images.githubusercontent.com/10931297/151868564-84d9e021-b701-4301-ad7d-87d7f8af4bc4.png)

```suggestion
You can specify the verion of ``dbt-adapter`` used in `pre-commit`_,
for example:

.. code-block:: yaml

   additional_dependencies : ['dbt-bigquery==1.0.0', 'sqlfluff-templater-dbt']
```

Which looks like this:

![image](https://user-images.githubusercontent.com/10931297/151868831-d1c1ed52-8d8c-4f85-9cdb-9c7ca6b2ced2.png)

It's true it does overflow with a scrollbar but so does the section above. I think this looks better.

WDYT?

Reply:
Agreed. I did not think about doing a code-block like that.